app/view/toolbar/MinimizedWindows.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * This class is the toolbar of cpsi mapview application that contains
+ * minimized windows. After a window was resized, the toolbar will be visible.
+ * If a window was minimized without adding this toolbar in advance, the window
+ * will minimize into nowhere.
+ *
+ * @class CpsiMapview.view.toolbar.MinimizedWindows
+ */
+Ext.define('CpsiMapview.view.toolbar.MinimizedWindows', {
+    extend: 'Ext.toolbar.Toolbar',
+    xtype: 'cmv_minimized_windows',
```

Comment:
Adding `_toolbar`, see above...

Reply:
solved in https://github.com/compassinformatics/cpsi-mapview/pull/179/commits/0b106a127d97601d29ea31d149448e09874ec88e